doc/releases/emperor.rst

```diff
@@ -194,7 +194,7 @@ Upgrading from v0.67 Dumpling
   2012) and store radosgw data, the auto-upgrade from TMAP to OMAP
   objects has been disabled.  Before upgrading, make sure that any
   buckets created on pre-argonaut releases have been modified (e.g.,
-  by PUTing and then DELETEing an object from each bucket).  Any
+  by PUTting and then DELETEing an object from each bucket).  Any
```

Comment:
I'd keep the Emperor release notes untouched.

Reply:
Fixed.